test/unit/views/payment-sheet-views/apple-pay-view.js

```diff
@@ -153,6 +158,25 @@ describe('ApplePayView', function () {
       }.bind(this));
     });
 
+    it('reports developer error when canMakePaymentsWithActiveCard returns false in sandbox mode', function (done) {
+      var configuration = fake.configuration();
+
+      configuration.gatewayConfiguration.environment = 'sandbox';
+      this.fakeClient.getConfiguration.returns(configuration);
+
+      this.sandbox.stub(console, 'error');
+      global.ApplePaySession.canMakePaymentsWithActiveCard = this.sandbox.stub().resolves(false);
+
+      this.view.initialize().then(function () {
+        this.view.model.reportError = function (err) {
+          expect(err).to.equal('developerError');
+          expect(console.error).to.be.calledWith('Could not find an active card. This may be because your Apple Pay Session was created in the Sandbox environment and your iCloud account is a production account. Log in to a Sandbox iCloud account to test this flow. If you are already logged into an iCloud Sandbox account, then add a card to your wallet.'); // eslint-disable-line no-console
```

Comment:
```suggestion
          expect(console.error).to.be.calledWith('Could not find an active card. This may be because you\'re using a production iCloud account in a sandbox Apple Pay Session. Log in to a Sandbox iCloud account to test this flow, and add a card to your wallet. For additional assistance, visit  https://help.braintreepayments.com'); // eslint-disable-line no-console
```